docs/reference/api/docker_remote_api_v1.21.md

```diff
@@ -2041,6 +2041,7 @@ Query Parameters:
   -   `event=<string>`; -- event to filter
   -   `image=<string>`; -- image to filter
   -   `container=<string>`; -- container to filter
+  -   `label=<string>`; -- image and container label to filter
```

Comment:
not your fault, but now that this list is getting longer, would you mind putting the list in alphabetical order?